app/src/main/java/eu/kanade/domain/extension/anime/interactor/CreateAniyomiExtensionRepo.kt

```diff
@@ -24,3 +35,4 @@ class CreateAnimeExtensionRepo(private val preferences: SourcePreferences) {
 
 const val OFFICIAL_ANIYOMI_REPO_BASE_URL = "https://raw.githubusercontent.com/aniyomiorg/aniyomi-extensions/repo"
 private val repoRegex = """^https://.*/index\.min\.json$""".toRegex()
+private val githubRepoRegex = """https://github\.com/[^/]+/[^/]+/blob/(?:[^/]+/)+index\.min\.json""".toRegex()
```

Comment:
```suggestion
private val githubRepoRegex = """https://github\.com/[^/]+/[^/]+/blob/(?:[^/]+/)+index\.min\.json$""".toRegex()
```
`...index.min.jsonbruh` would also match without the `$`